flexget/plugins/output/decompress.py

```diff
@@ -28,13 +28,14 @@ def open_archive_entry(entry):
     archive = None
 
     try:
-        archive_path = entry['location']
-        if not archive_path:
-            raise ValueError('Entry does not appear to represent a local file.')
+        archive_path = entry.get('location')
 
-        archive = archiveutil.open_archive(archive_path)
-    except (KeyError, ValueError):
-        log.error('Entry does not appear to represent a local file.')
+        if not os.path.exists(archive_path):
```

Comment:
This will raise an exception if archive_path is None

Reply:
Addressed with my latest commit

Reply:
Wouldn't it be better to reverse the if-else?
```
if not archive_path:
elif not os.path.exists
```

Reply:
Not a bad idea. Let me change it.